.github/quarkus-bot.yml

```diff
@@ -439,3 +439,6 @@ triage:
     - labels: [kind/extension-proposal]
       title: quarkiverse
       notify: [gastaldi, gsmet, aloubyansky]
+  qe:
+    - rsvoboda
+    - mjurc
```

Comment:
I know that's not what I recommended but in retrospect, I would prefer if we follow the same conventions we use for the other triage rules.

So that would be:
```
  qe:
    notify: [rsvoboda, mjurc]
```
(and the bot should also be adjusted)

Sorry about that.